README.md

```diff
@@ -1,6 +1,86 @@
-# silk-paper
-Package for the automatic generation of REST documentation based on the OpenApi specification. Key name: Silk
+# silk-paper-node
+Package for the automatic generation of REST documentation based on the OpenApi specification.
+
+## Installing
+Install with `npm i @wolox/silk-paper`.
+
+## Usage
+* In `app.js`:
+We are using [`swagger-ui-express`](https://www.npmjs.com/package/swagger-ui-express) package for render the documentation, so first you have to install it.
+```
+const swaggerUi = require('swagger-ui-express');
+const silkPaper = require('@wolox/silk-paper');
+
+app.use('/docs', swaggerUi.serve, swaggerUi.setup(silkPaper.buildDocs()));
+``` 
+
+* In `test_file.js`:
+```
+const server = require('./app.js');     //This is a server instance.
+const SilkPaper = require('@wolox/silk-paper');
+const docs = new SilkPaper(server, {});
+``` 
+
+And in the test case that we want to document adding the following sentence:
+```
+docs.genDocs(request, {});
+```
+
+### Supported extensions: 
+This should be added as option in the constructor and the package accepts two kind of type files:
+* .json
+* .yml
+
+### Options
+* For constructor:
+``` 
+{
+  docsDir: 'custom documentatio path',
+  fileType: 'json',
+  openApiVersion: '3.0.0'
+}
+```
+* For `genDocs` function:
+``` 
+{
```

Comment:
Put the function here as well, I know it may not seem necessary but is more telling if we put it